testing/regress/ecl/indexread_keyed.ecl

```diff
@@ -23,8 +23,8 @@
 //version multiPart=true,useLocal=true,forceRemoteKeyedLookup=true,forceRemoteKeyedFetch=true
 
 import ^ as root;
-multiPart := #IFDEFINED(root.multiPart, true);
-useLocal := #IFDEFINED(root.useLocal, false);
+multiPart := #IFDEFINED(root.multiPart, false);
```

Comment:
Not sure what this change is for?

Reply:
It was a change made while testing.  It only affects the defaults, but I'll regress it to remove the confusion.